### ISSUE TITLE
Explain that time trigger should be done in UTC

### ISSUE THREAD

**Issue by ruudk** [home-assistant/frontend#22394]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

It turns out that triggering on time should be defined in UTC. This was not clear.

![Screenshot 2024-10-16 at 10 01 24@2x](https://github.com/user-attachments/assets/5add4610-88ae-420e-b732-2e96e5e95a3a)

### Describe the behavior you expected

Add a label to explain this.

### Steps to reproduce the issue

.

### What version of Home Assistant Core has the issue?

2024.10.2

**Comment by karwosts** [comment]
I don't believe that is the case, it should be in the time zone of your server.

**Comment by ruudk** [comment]
Ok, I think the problem is that the time trigger just triggers at random times.

I schedule at 7 but get my notification at 8:44
